com.unity.render-pipelines.universal/Runtime/DeferredLights.cs

```diff
@@ -660,6 +701,82 @@ public void OnCameraCleanup(CommandBuffer cmd)
                 m_stencilVisLightOffsets.Dispose();
         }
 
+        public static StencilState OverwriteStencil(StencilState s, int stencilWriteMask)
```

Comment:
Add API docs or consider if it should be internal.

Reply:
Made internal.